test/TestConfig/settings.mk

```diff
@@ -0,0 +1,247 @@
+# cmdlinetester jars
+#######################################
+CMDLINETESTER_JAR   =$(Q)$(JVM_TEST_ROOT)$(D)cmdline_options_tester$(D)cmdlinetester.jar$(Q)
+CMDLINETESTER_RESJAR=$(Q)$(JVM_TEST_ROOT)$(D)cmdline_options_testresources$(D)cmdlinetestresources.jar$(Q)
+
+#######################################
+# testng report dir
+#######################################
+ifndef UNIQUEID
+	GETID := $(TEST_ROOT)$(D)TestConfig$(D)scripts$(D)testKitGen$(D)resultsSummary$(D)getUniqueId.pl
+	export UNIQUEID := $(shell perl $(GETID) -v)
+endif
+TESTOUTPUT := $(TEST_ROOT)$(D)TestConfig$(D)test_output_$(UNIQUEID)
+REPORTDIR = $(Q)$(TESTOUTPUT)$(D)$@$(Q)
+
+#######################################
+# TEST_STATUS
+#######################################
+TEST_STATUS=if [ $$? -eq 0 ] ; then $(ECHO) $(Q)$@$(Q)$(Q)_PASSED$(Q); else $(ECHO) $(Q)$@$(Q)$(Q)_FAILED$(Q); fi
+ifneq ($(DEBUG),)
+$(info TEST_STATUS is $(TEST_STATUS))
+endif
+TEST_SKIP_STATUS=$@_SKIPPED
+
+#######################################
+# include configure makefile
+#######################################
+ifndef MACHINE_MK
+-include $(JVM_TEST_ROOT)$(D)TestConfig$(D)machineConfigure.mk
+else
+-include $(MACHINE_MK)$(D)machineConfigure.mk
+endif
+
+#######################################
+# include sub makefile
+#######################################
+-include $(JVM_TEST_ROOT)$(D)TestConfig$(D)extraSettings.mk
+
+#######################################
+# generic target
+#######################################
+_TESTTARGET = $(firstword $(MAKECMDGOALS))
+TESTTARGET = $(patsubst _%,%,$(_TESTTARGET))
+
+SUBDIRS_TESTTARGET = $(SUBDIRS:%=$(TESTTARGET)-%)
+
+$(SUBDIRS_TESTTARGET):
+	@if [ -d $(@:$(TESTTARGET)-%=%) ]; then \
+		$(MAKE) -C $(@:$(TESTTARGET)-%=%) -f autoGen.mk $(TESTTARGET); \
+	fi
+
+$(TESTTARGET): $(SUBDIRS_TESTTARGET)
+
+_$(TESTTARGET): setup_$(TESTTARGET) rmResultFile $(TESTTARGET) resultsSummary
+	@$(ECHO) $@ done
+
+.PHONY: _$(TESTTARGET) $(TESTTARGET) $(SUBDIRS) $(SUBDIRS_TESTTARGET)
+
+.NOTPARALLEL: _$(TESTTARGET) $(TESTTARGET) $(SUBDIRS) $(SUBDIRS_TESTTARGET)
+
+setup_%:
+	@$(ECHO)
+	@$(ECHO) Running make $(MAKE_VERSION)
+	@$(ECHO) set TEST_ROOT to $(TEST_ROOT)
+	@$(ECHO) set JAVA_VERSION to $(JAVA_VERSION)
+	@$(ECHO) set JCL_VERSION to $(JCL_VERSION)
+	@$(ECHO) set JAVA_BIN to $(JAVA_BIN)
+	@$(ECHO) set SPEC to $(SPEC)
+	@$(ECHO) Running $(TESTTARGET)
+	$(JAVA_COMMAND) -version
+
+ifndef JCL_VERSION
+export JCL_VERSION:=latest
+else
+export JCL_VERSION:=$(JCL_VERSION)
+endif
+
+# Define the EXCLUDE_FILE to be used for temporarily excluding failed tests.
+# This macro is used in /test/Utils/src/org/openj9/test/util/IncludeExcludeTestAnnotationTransformer
+ifndef EXCLUDE_FILE
+	export EXCLUDE_FILE:=$(JVM_TEST_ROOT)$(D)TestConfig$(D)resources$(D)excludes$(D)$(JCL_VERSION)_exclude_$(JAVA_VERSION).txt
+endif
+
+#######################################
+# failed target
+#######################################
+FAILEDTAGETS = $(TESTOUTPUT)$(D)failedtargets.mk
```

Comment:
would love to see this spelling mistake fixed, should be FAILEDTARGETS with an R